gettingStarted.css

```
@@ -0,0 +1,123 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,900&display=swap');
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Lato', sans-serif;
+}
+
+.body {
+    height: 70rem;
+}
+
+.page-align {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    height: 20rem;
+}
+
+footer {
+    width: 100%;
+    text-align: center;
+    background-color: black;
+    color: white;
+    padding: 1rem 0;
+}
```

```suggestion
footer {
    width: 100%;
    text-align: center;
    background-color: black;
    color: white;
    padding: 1rem 0;
    position: fixed;
    bottom: 0;
}
```
This will make the footer fixed to bottom